tests/app/test_model.py

```diff
@@ -52,15 +50,19 @@ def test_should_build_service_guest_list_from_mobile_number(mobile_number):
 @pytest.mark.parametrize("email_address", ["test@example.com"])
 def test_should_build_service_guest_list_from_email_address(email_address):
     service_guest_list = ServiceGuestList.from_string(
-        "service_id", EMAIL_TYPE, email_address
+        "service_id", RecipientType.EMAIL, email_address
     )
 
     assert service_guest_list.recipient == email_address
 
 
 @pytest.mark.parametrize(
     "contact, recipient_type",
-    [("", None), ("07700dsadsad", MOBILE_TYPE), ("gmail.com", EMAIL_TYPE)],
+    [
+        ("", None),
+        ("07700dsadsad", RecipientType.MOBILE),
+        ("gmail.com", RecipientType.EMAIL),
```

Comment:
I realize this is test and fake data, I just want to make sure we're clear on how it's being used!